.gitignore

```diff
@@ -118,5 +118,6 @@ dist
 # Telemetry data
 /telemetry
 
-# Extension bundles
-*.vsix
```

Comment:
Note that the reason I deleted this line is because it's a duplicate of line 75 which I added in #11:

https://github.com/nanofuzz/nanofuzz/blob/d59393394eb6ddf335f72fe44ab47a439354d3a3/.gitignore#L74-L75